---
layout: default
title: Publications and Funding
permalink: /Publications/
---
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>


### Publications (career citation metrics: 240 citations, h-index= 8, i10-index= 8)

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1101/2020.04.17.046896"></div>

**Henson, M.W.**, Lanclos, V.C., Pitre, D.M., Weckhorst, J.L., Lucchesi, A.M., Cheng, C., Temperton, B. and Thrash, C. (2020). [Expanding the diversity of bacterioplankton isolates and modeling isolation efficacy with large scale dilution-to-extinction cultivation](https://doi.org/10.1101/2020.04.17.046896){:target="_blank"}. *Submitted to AEM*. *bioRxiv*. 

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/MRA.01492-19"></div>

**Henson, M.W.**, Guidry, M.E., Carnes, M.K., Thrash, J.C. (2020). [Draft genome sequence of the novel coastal bacterium LSUCC0115 from the MWH-UniPo clade, Order Burkholderiales, Class Betaproteobacteria](https://doi.org/10.1128/MRA.01492-19){:target="_blank"}. *Microbiology Resource Announcements* 9(2), pp.e01492-19

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/genomeA.01231-16"></div>

Lanclos, V.C., **Henson, M.W.**, Dorion, C., Thrash, J.C. (2019). [Draft genome of strain LSUCC0057, a member of the SAR92 clade of Gammaproteobacteria](https://doi.org/10.1128/genomeA.01231-16){:target="_blank"}. *Microbiology Resource Announcements* 8(25), pp.e00599-19.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/MRA.00521-19"></div>

Coelho, J.T., **Henson, M.W.**, Thrash, J.C. (2019). [Draft genome sequence of strain LSUCC0112, a Gulf of Mexico representative of the Oligotrophic Marine Gammaproteobacteria](https://doi.org/10.1128/MRA.00521-19){:target="_blank"}. *Microbiology resource announcements* 8(27), pp.e00521-19.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/MRA.00521-19"></div>

Learman, D.R., Ahmand, Z., Brookshier, A., **Henson, M.W.**, Hewitt, V., Lis, A., Morrison, C., Robinson, A., Todaro, E., Wologo, E., Wynne, S., Alm, E.W., Kourtev, P.S., Santo Domingo, J.W. (2019). [Comparative genomics of 16 Microbacterium spp. that tolerate multiple heavy metals and antibiotics](https://doi.org/10.7717/peerj.6258){:target="_blank"}. *PeerJ* 6, pp.e6258 

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1101/261727"></div>

Craine, J., **M. W. Henson**, J. C. Thrash, J. Hanssen, G. Spooner, P. Fleming, M. Pukonen, F. Stahr, S. Spaulding,
and N. Fierer. (2018). [Environmental DNA reveals the structure of phytoplankton assemblages along a 2900-km transect
 in the Mississippi River](https://doi.org/10.1101/261727){:target="_blank"}. *bioRxiv*

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1101/091512"></div>

**Henson M.W**., Hanssen J, Spooner G, Flemming P, Pukonen M, Stahr F, Thrash, J.C. (2018). [Microbial regime changes
and indicators of eutrophication on the Mississippi River identified via a human-powered 2900 km transect](https://doi.org/10.1101/091512){:target="_blank"}. *Limnology and Oceanography* 63(5), pp.1837-1855.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1038/s41396-018-0092-2"></div>

**Henson, M.W.**, Lanclos, V.C., Faircloth, B.C., Thrash, J.C. (2018). [Cultivation and genomics of the first freshwater SAR11 (LD12) isolate](https://www.nature.com/articles/s41396-018-0092-2){:target="_blank"}.
  *ISME Journal* 12, pp.1846–1860.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/genomeA.01231-16"></div>

Lanclos, V.C., **Henson, M.W.**, Pitre, D.M. and Thrash, J.C. (2016). [Draft genome sequence of strain LSUCC0135, an early diverging member of the order *Methylophilales* in the phylum Betaproteobacteria](http://genomea.asm.org/content/4/6/e01231-16.full){:target="_blank"}. *Genome announcements* 4(6), pp.e01231-16.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/mSphere.00028-16"></div>

**Henson, M.W.**, Pitre, D.M., Weckhorst, J.L., Lanclos, V.C., Webber, A.T., Thrash, J.C. (2016). [Artificial seawater
media facilitates cultivating members of the microbial majority from the Gulf of Mexico](http://msphere.asm.org/content/1/2/e00028-16){:target="_blank"}. *mSphere* 1(2), pp.e00028-16 **Editor’s pick**

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.3389/fmicb.2016.00284"></div>

Learman, D.R., **Henson, M.W.**, Thrash, J.C., Temperton, B., Brannock, P., Mahon, A., Halanych, K.H. (2016)
[Biogeochemical and microbial variation across 5500 km of Antarctic surface sediment implicates organic matter as a
driver of benthic community structure](https://dx.doi.org/10.3389/fmicb.2016.00284){:target="_blank"}. *Frontiers in Microbiology: Aquatic Microbiology* 7, pp.284.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1007/s13213-015-1189-8"></div>

Ni, C., Horton, D. J., Rui, J., **Henson, M.W.**, Jiang, Y., Huang, X., & Learman, D. R. (2016). [High concentrations of bioavailable
heavy metals impact freshwater sediment microbial communities](https://link.springer.com/article/10.1007/s13213-015-1189-8){:target="_blank"}. *Annals of Microbiology* 66, pp.1003–1012.

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.7717/peerj.1395"></div>

**Henson M.W.**, Santo Domingo J.W., Kourtev P.S., Jensen R.V., Dunn J.A., Learman D.R. (2015). [Metabolic and genomic
analysis elucidates strain-level variation in Microbacterium spp. isolated from chromate contaminated sediment](https://dx.doi.org/10.7717/peerj.1395){:target="_blank"}. *PeerJ* 3, pp.e1395

<div style="float: right; padding-left: 15px" class='altmetric-embed' data-badge-type='donut' data-link-target='_blank' data-doi="10.1128/AEM.02802-12"></div>

Ryu, H., **Henson, M.**, Elk, M., Toledo-Hernandez, C., Griffith, J., Blackwood, D., Noble, R., Gourmelon, M.,
Glassmeyer, S., and Santo Domingo, J. (2013). [Development of quantitative PCR assays targeting 16S rRNA gene of
*Enterococcus* spp. and their application to the identification of *Enterococcus* species in environmental samples](http://aem.asm.org/content/79/1/196.full){:target="_blank"}. *Applied Environmental Microbiology*. 79(1), pp.196-204.

### Funding

*2017 - 2018*	Lerner Gray Grant, “Examining the functional differences of two important SAR11 subclades and its insights on how
salinity changes may alter global biogeochemical cycles.” American Natural History Museum, $2500

*2015 - 2016*	LEEC Research Grant, “Assessment of wetlands on the Louisiana coast using baseline microbial community structure
and developing microbial indices”, LA Department of Wildlife & Fisheries, $1138
